src/scripts/components/utils/localStorage.js

```diff
@@ -0,0 +1,25 @@
+export default class LocalStorage {
+  saveData = ({ data }) => {
+    window.localStorage.setItem('data', JSON.stringify(data));
+  };
+
+  /**
+   * Gets all loaded user data from previous session.
```

Comment:
[OPTIONAL]
- Keep your project professional and remove unused comments in the code.